src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -238,6 +239,79 @@ public String getQuestionAdditionalInfoHtml(int questionNumber,
+        if (!question.showResponsesTo.contains(FeedbackParticipantType.RECEIVER)
+                || !question.showResponsesTo.contains(FeedbackParticipantType.OWN_TEAM_MEMBERS)) {
+            return "";
+        }
+
+        List<FeedbackResponseAttributes> actualResponses = getActualResponses(question, bundle);
+
+        Map<String, List<Integer>> recipientRanks = generateOptionRanksMapping(actualResponses);
+
+        boolean isRecipientTypeTeam = question.recipientType == FeedbackParticipantType.TEAMS
+                                              || question.recipientType == FeedbackParticipantType.OWN_TEAM;
+
+        String currentUserTeam = bundle.getTeamNameForEmail(studentEmail);
+        String currentUserIdentifier = isRecipientTypeTeam ? currentUserTeam : studentEmail;
+        List<Integer> ranksReceived = recipientRanks.get(currentUserIdentifier);
+        if (ranksReceived == null) {
+            return "";
+        }
+
+        StringBuilder fragments = new StringBuilder();
+        Map<String, Integer> recipientOverallRank = generateNormalizedOverallRankMapping(recipientRanks);
+
+        Map<String, List<Integer>> recipientRanksExcludingSelf = getRecipientRanksExcludingSelf(actualResponses);
+
+        Map<String, Integer> recipientOverallRankExceptSelf =
+                generateNormalizedOverallRankMapping(recipientRanksExcludingSelf);
+
+        Map<String, Integer> recipientSelfRanks = generateSelfRankForEachRecipient(actualResponses);
+
+        String fragmentTemplateToUse = FormTemplates.RANK_RESULT_STATS_RECIPIENTFRAGMENT;
+        String templateToUse = FormTemplates.RANK_RESULT_RECIPIENT_STATS;
+        String ranksReceivedAsString = getListOfRanksReceivedAsString(ranksReceived);
+        String overallRank = Integer.toString(recipientOverallRank.get(currentUserIdentifier));
+        String name = bundle.getNameForEmail(currentUserIdentifier);
+
+        String overallRankExceptSelf = recipientOverallRankExceptSelf.containsKey(currentUserIdentifier)
+                ? Integer.toString(recipientOverallRankExceptSelf.get(currentUserIdentifier)) : "-";
+        String selfRank = recipientSelfRanks.containsKey(currentUserIdentifier)
+                ? Integer.toString(recipientSelfRanks.get(currentUserIdentifier)) : "-";
+
+        fragments.append(Templates.populateTemplate(fragmentTemplateToUse,
+                Slots.RANK_OPTION_VALUE, SanitizationHelper.sanitizeForHtml(name),
+                Slots.TEAM, SanitizationHelper.sanitizeForHtml(currentUserTeam),
+                Slots.RANK_RECIEVED, ranksReceivedAsString,
+                Slots.RANK_SELF, selfRank,
+                Slots.RANK_OVERALL, overallRank,
+                Slots.RANK_EXCLUDING_SELF_OVERALL, overallRankExceptSelf));
+
+        String statsTitle = isRecipientTypeTeam ? "Summary of responses received by your team"
+                : "Summary of responses received by you";
```

Comment:
`Summary of responses you received`？

Reply:
Numerical questions statistics has `Summary of responses received by you`. Title should be consistent?